CONTRIBUTING.md

```diff
@@ -114,62 +184,76 @@ Closes #123
 ```
 
 Additionally, follow these guidelines for the message content:
-- Use the present tense ("Add feature" not "Added feature")
-- Use the imperative mood ("Move cursor to..." not "Moves cursor to...")
-- Limit the first line to 72 characters or less
-- Reference issues and pull requests liberally after the first line
 
-Example:
-```
-Add compression support for NFT metadata
+-   Use the present tense ("Add feature" not "Added feature").
+-   Use the imperative mood ("Move cursor to..." not "Moves cursor to...").
+-   Limit the first line to 72 characters or less.
+-   Reference issues and pull requests liberally after the first line.
+
+### Code Style
 
-- Implement ZSTD compression for metadata storage
-- Add decompression utilities
-- Update tests to cover compression/decompression
-- Fixes #123
+We use `rustfmt` and `clippy` to enforce a consistent code style. Run these tools before submitting a pull request:
+
+```bash
+cargo +nightly fmt
+cargo clippy --all -- -D warnings
 ```
 
+The `rustfmt.toml` file in the repository defines the formatting rules. We use a minimal `.clippy.toml` that allows deprecated code, mirroring the CI configuration.
+
 ### Continuous Integration
 
 All branches must pass our CI pipeline before being merged. The CI pipeline includes:
-- Code style checks
-- Unit tests
-- Integration tests
-- Build verification
+
+-   **Code Style Checks:** `rustfmt` (nightly) and `clippy`.
+-   **Unit Tests:** `cargo test`.
+-   **Integration Tests:** `cargo test -F integration_tests`.
```

Comment:
i think the unit tests here can be omitted as integration tests are a superset of unit tests